kube/stateful_set.go

```diff
@@ -68,7 +68,14 @@ func getVolumeClaims(role *model.Role, createHelmChart bool) []helm.Node {
 		}
 
 		meta := helm.NewMapping("name", volume.Tag)
-		meta.Add("annotations", helm.NewMapping(VolumeStorageClassAnnotation, storageClass))
+		annotationList := helm.NewMapping()
+		annotationList.Add(VolumeStorageClassAnnotation, storageClass)
+		if volume.Annotations != nil || len(volume.Annotations) > 0 {
```

Comment:
I don't think you need the `if` anymore, since (IIRC) `range nil {…}` just runs zero iterations. Even if it didn't, you don't need the `volume.Annotations != nil` (because `len(nil)` is zero).  And even if needed _that_, you wouldn't want an `||`, but a `&&` :)

Reply:
terrible that `||`, thanks for spotting that mistake.